src/base/bittorrent/session.cpp

```diff
@@ -4007,10 +4018,8 @@ void Session::handleTorrentRemovedAlert(const lt::torrent_removed_alert *p)
 
     if (m_removingTorrents.contains(infoHash)) {
         const RemovingTorrentData tmpRemovingTorrentData = m_removingTorrents[infoHash];
-        if (!tmpRemovingTorrentData.requestedFileDeletion) {
```

Comment:
Why did you remove this condition?

Reply:
https://github.com/qbittorrent/qBittorrent/pull/11241#discussion_r324883290

Reply:
>This "if" isn't needed because it's always "Torrent" here

Are you sure? Did you test it?

Reply:
I see the problem now, when the `if ()` here is omitted it *might* produce duplicated/confusing log message, that is this message here and the ones in `Session::handleTorrentDeletedAlert` & `Session::handleTorrentDeleteFailedAlert`. It should be tested.

Reply:
>It should be tested.

It should be reverted here since it's not part of this job.

Reply:
It should be reverted anyway... didn't know that this alert was posted when also deleting the files and as a consequence of removing this `if ()`, the torrent's hash would be removed from `m_removingTorrents` and the stuff in `handleTorrentDeletedAlert/handleTorrentDeleteFailedAlert` would never be executed. Whew thanks for catching this.

Reply:
Reverted.